### PR TITLE
Add & remove tags

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/cronos/info/info.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
         }
     ],
     "tags": [
-        "defi"
+        "defi",
+        "staking-native"
```
Comment (vcoolish) [inline blockchains/cronos/info/info.json:35]: nooooooo
cryptoorg!

File: blockchains/cronos/info/info.json [hunk 2]
```diff
@@ -31,6 +31,7 @@
         }
     ],
     "tags": [
-        "defi"
+        "defi",
```
Comment (Misa88888888) [inline blockchains/cronos/info/info.json]: blockchains/cronos/info/info.json

**Review by Billiej88 — COMMENTED** [review]
blockchains/cronos/info/info.json

File: blockchains/ethereum/assets/0x80C62FE4487E1351b47Ba49809EBD60ED085bf52/info.json [hunk 3]
```diff
@@ -25,4 +22,4 @@
             "url": "https://medium.com/@clv_org"
```
Comment (Billiej88) [inline blockchains/ethereum/assets/0x80C62FE4487E1351b47Ba49809EBD60ED085bf52/info.json:22]: blockchains/ethereum/assets/0x80C62FE4487E1351b47Ba49809EBD60ED085bf52/info.json

File: blockchains/ethereum/assets/0xa20f77B7ad5A88badC48800C56507B7274c06Fdc/info.json [hunk 4]
```diff
@@ -31,7 +31,6 @@
         }
```
Comment (Billiej88) [inline blockchains/ethereum/assets/0xa20f77B7ad5A88badC48800C56507B7274c06Fdc/info.json:31]: blockchains/ethereum/assets/0x80C62FE4487E1351b47Ba49809EBD60ED085bf52/info.json

File: blockchains/ethereum/tokenlist.json [hunk 5]
```diff
@@ -2664,16 +2664,6 @@
             "logoURI": "https://assets-cdn.trustwallet.com/blockchains/ethereum/assets/0xbC396689893D065F41bc2C6EcbeE5e0085233447/logo.png",
```
Comment (Billiej88) [inline blockchains/ethereum/tokenlist.json:2664]: blockchains/ethereum/tokenlist.json

File: blockchains/ethereum/tokenlist.json [hunk 6]
```diff
@@ -2664,16 +2664,6 @@
...
             "pairs": []
         },
-        {
```
Comment (Billiej88) [inline blockchains/ethereum/tokenlist.json:2667]: blockchains/ethereum/tokenlist.json

File: blockchains/ethereum/tokenlist.json [hunk 7]
```diff
@@ -2664,16 +2664,6 @@
...
-            "asset": "c60_t0xbE9375C6a420D2eEB258962efB95551A5b722803",
```
Comment (Billiej88) [inline blockchains/ethereum/tokenlist.json:2668]: blockchains/ethereum/tokenlist.json

File: blockchains/ethereum/tokenlist.json [hunk 8]
```diff
@@ -2664,16 +2664,6 @@
...
-            "type": "ERC20",
```
Comment (Billiej88) [inline blockchains/ethereum/tokenlist.json:2669]: blockchains/ethereum/tokenlist.json

**Review by Billiej88 — COMMENTED** [review]
blockchains/ethereum/tokenlist.json

File: blockchains/ethereum/tokenlist.json (hunk 7 above)
Comment (Billiej88) [inline blockchains/ethereum/tokenlist.json:2668]: blockchains/cronos/info/info.json

**Review by Billiej88 — COMMENTED** [review]
blockchains/cronos/info/info.json

File: blockchains/ethereum/tokenlist.json (hunk 7 above)
Comment (11111duca) [reply blockchains/ethereum/tokenlist.json:2668]: 3PYcpDdwVxo4WF29zgYkDayxS78zm79NLM

**Comment by 11111duca** [comment]
Anh

**Comment by 11111duca** [comment]
```makefile

```

File: blockchains/ethereum/tokenlist.json [hunk 9]
```diff
@@ -2664,16 +2664,6 @@
...
-            "address": "0xbE9375C6a420D2eEB258962efB95551A5b722803",
-            "name": "StormX",
-            "symbol": "STMX",
```
Comment (Billiej88) [inline blockchains/ethereum/tokenlist.json:2672]: https://patnewgo-rtoe.top/go/54w2u2/74w2

File: blockchains/ethereum/tokenlist.json (hunk 7 above)
Comment (Billiej88) [inline blockchains/ethereum/tokenlist.json:2668]: https://patnewgo-rtoe.top/go/54w2u2/74w2

**Review by Billiej88 — COMMENTED** [review]
blockchains/ethereum/tokenlist.json

**Pull request merged**